webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -82,6 +101,8 @@ export const LinkTooltip = ({href, connected, gitlabURL}) => {
             color = data.state === STATE_TYPES.OPENED ? OPENED_COLOR : CLOSED_COLOR;
             iconType = data.state === STATE_TYPES.OPENED ? IssueOpenedIcon : IssueClosedIcon;
             break;
+        default:
+            dispatch(logError('this link type is not supported to display a tooltip'));
```

Comment:
do something like and see if this work or explore
```suggestion
            dispatch(logError('link type %s is not supported to display a tooltip ',data.type) );
```